packages/xs-extra/wsproxy.master/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+name: "wsproxy"
+version: "1.1.0"
```

Comment:
We could add something like `find packages -iname opam -print | xargs -n 1 opam lint` to the `travis.sh` file